File: Transpose two strings in an array.js
function transposeTwoStrings (array) {
	const long = array[0].length>=array[1].length?array[0]:array[1];
	return [...long].map((_,i) => (array[0][i]||' ')+' '+(array[1][i]||' ')).join('\n');
}

File: Transpose two strings in an array.py
from itertools import zip_longest
def transpose_two_strings(arr):
    result = []
    for i,j in zip_longest(arr[0],arr[1]):
        tmp = ""
        if i is None:
            tmp+=" "
        else:
            tmp+=i
        tmp+=" "
        if j is None:
            tmp+=" "
        else:
            tmp+=j
        result.append(tmp)
    return "\n".join(result)
        
        
